PRChecker/Shared/Common Views/TagListView.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  TagListView.swift
+//  PRChecker
+//
+//  Created by Chen Yuhan on 2021/11/14.
+//
+
+import SwiftUI
+
+struct TagListView: View {
+    var tagViews: [TagView]
+
+    @State private var totalHeight: CGFloat = 0
+
+    // Ref: https://stackoverflow.com/questions/62102647/swiftui-hstack-with-wrap-and-dynamic-height
+    var body: some View {
+        var width: CGFloat = 0
+        var height: CGFloat = 0
+
+        return VStack {
+            GeometryReader { geometry in
+                let containerWidth = geometry.size.width
+
+                ZStack(alignment: .topLeading) {
+                    ForEach(self.tagViews, id: \.self) { tagView in
```

Comment:
I guess we could have used `text` as well, but `self` works too. (I made a bug with this that I will be fixing in a later PR 😛 )

Reply:
Curious what bug it is 😂

Reply:
The `id` on the PRList. If the same PR is assigned to you + watchers or multiple watchers, it won't be unique and SwiftUI will crash.